databricks/koalas/generic.py

```diff
@@ -586,7 +613,7 @@ def to_csv(
         partition_cols: Optional[Union[str, List[str]]] = None,
         index_col: Optional[Union[str, List[str]]] = None,
         **options
-    ):
+    ) -> None:
```

Comment:
`Optional[str]`? `to_csv` will return string when `path is None`.

Reply:
Good catch! Updated.